src/lib/generateCrumbs.ts

```diff
@@ -212,4 +212,8 @@ export const processParts = (
       // Filter out the last element if excludeCurrentPage is true
       return !(excludeCurrentPage && part.isLast);
     });
+
+    // Set the last element as the previous last element may have been excluded
+    if(processedParts.length > 0) processedParts[processedParts.length-1].isLast = true;
+    return processedParts
```

Comment:
Works well, but I would like to adjust the condition a bit.

```suggestion
  // Remove the separator for the last part if excludeCurrentPage is true
  if (excludeCurrentPage && processedParts.length > 0) {
    // Set the last element as the previous last element
    processedParts[processedParts.length - 1].isLast = true;
 }
  
    return processedParts
```